circus/_patch.py

```diff
@@ -33,15 +32,3 @@ def _delete(self):  # NOQA
                     raise
```

Comment:
I think this patch also includes some things related to old python versions. I don't think we can completely drop the patch though, as the issue doesn't look to be fixed upstream.

Reply:
Do you think removing only the part about 2.7 is enough? I read through the issues you mentioned and yes, they are still open, so I am not sure if there is anything else we can do here.

Reply:
Let's keep it as it is.